index.html

```diff
@@ -0,0 +1,125 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Home Assistant</title>
+
+    <link rel='manifest' href='/manifest.json'>
+    <link rel='icon' href='/static/icons/favicon.ico'>
+    <link rel='apple-touch-icon' sizes='180x180'
+          href='/static/icons/favicon-apple-180x180.png'>
+    <link rel="mask-icon" href="/static/icons/mask-icon.svg" color="#3fbbf4">
+    {% if not dev_mode %}
+      <link rel='preload' href='/home-assistant-polymer/build/core.js' as='script'/>
+      {% for panel in panels.values() -%}
+        <link rel='prefetch' href='{{ panel.webcomponent_url }}'>
```

Comment:
{% if es6 %}
  <link rel='prefetch' href='{{ panel.webcomponent_url }}'>
{% else %}
  <link rel='prefetch' href='{{ panel.webcomponent_url_es5 }}'>
{% endif %}

Reply:
I'm going to remove it completely. It is already cached in the service worker so we should be good. From my traces it actually gets loaded before important assets. Removing it will give us a nice little speed bump 👍